translation/file_loader_test.go

```diff
@@ -2,38 +2,49 @@ package translation
 
 import (
 	"os"
+	"path/filepath"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/suite"
 
+	"github.com/goravel/framework/mocks/foundation"
 	"github.com/goravel/framework/support/env"
 	"github.com/goravel/framework/support/file"
 )
 
 type FileLoaderTestSuite struct {
 	suite.Suite
+	app *foundation.Application
 }
 
 func TestFileLoaderTestSuite(t *testing.T) {
-	assert.Nil(t, file.Create("lang/en/test.json", `{"foo": "bar", "baz": {"foo": "bar"}}`))
-	assert.Nil(t, file.Create("lang/en/another/test.json", `{"foo": "backagebar", "baz": "backagesplash"}`))
-	assert.Nil(t, file.Create("lang/another/en/test.json", `{"foo": "backagebar", "baz": "backagesplash"}`))
-	assert.Nil(t, file.Create("lang/en/invalid/test.json", `{"foo": "bar",}`))
+	executable, err := executablePath()
+	assert.NoError(t, err)
+
+	assert.Nil(t, file.Create(filepath.Join(executable, "lang/en/test.json"), `{"foo": "bar", "baz": {"foo": "bar"}}`))
```

Comment:
Looks better, but could you please add additional test cases for `filepath.Join(executable, ***)`, don't change the original test cases?